test/test-parse-statement.cpp

```diff
@@ -593,6 +606,19 @@ TEST_F(Test_Parse_Statement, if_with_else) {
                               "visit_variable_use",               // b
                           }));
   }
+
+  {
+    Spy_Visitor p = test_parse_and_visit_statement(
+        u8"if(x)\n\tif (y)\n\t\tz();\nelse\n\tw();"_sv,  //
+        u8"         ^^ Diag_Misleading_Braceless_If_Else_Indentation.if_span\n"_diag  //
+        u8"                           ^^^^ .else_span"_diag);
```

Comment:
This looks good, though I find `\t` a bit hard to read. Perhaps two spaces is a good substitute:
```suggestion
        u8"if(x)\n  if (y)\n    z();\nelse\n  w();"_sv,  //
        u8"         ^^ Diag_Misleading_Braceless_If_Else_Indentation.if_span\n"_diag  //
        u8"                           ^^^^ .else_span"_diag);
```
Eh, maybe not. We don't have great testing for diagnostics with multi-line code samples...